### ISSUE TITLE
the expression prompt mode only output one object?

### ISSUE THREAD

**Issue by AllenDun** [FoundationVision/GLEE#4]
in the hugging face demo, the expression prompt mode only output one object, even if there are multi same objects?

**Comment by 0xNOY** [comment]
Yes, in the demo, only one object is output in the expression prompt mode. To output multiple objects, you need to change the value of the variable `topK_instance` in [line 174 of app/app.py](https://github.com/FoundationVision/GLEE/blob/ac78c9d7c3fd00799d2320b234c41ba659085d08/app/app.py#L174C22-L174C22).

**Comment by AllenDun** [comment]
@0xNOY thanks very much. So I should change topk_instance to 10, e.g, which means top10?

**Comment by AllenDun** [comment]
could you please explain more details about the grouding mode and category mode in the tokenization stage? It seems that we can input multi categoires an once in a prompt, but only one expression in a prompt? Could both modes be the same?

**Comment by wjf5203** [comment]
@AllenDun Thank you for your interest! In our setting, if the input is an expression, the model will only find the single most relevant object, as topK=1 is a fixed parameter set by design. If you want to find multiple objects at the same time, you can try using the 'categories' -> 'custom-list' mode, which supports a list of arbitrary category names separated by commas. In this mode, you can freely set the TopK, and you can input multiple categories at once.
